osf_tests/factories.py

```diff
@@ -411,14 +411,15 @@ def _create(cls, target_class, project=None, is_public=False,
                 branched_from=project,
                 initator=user,
                 registration_schema=schema,
-                provider=provider
+                provider=provider,
             )
         auth = Auth(user=user)
         register = lambda: project.register_node(
             schema=schema,
             auth=auth,
             draft_registration=draft_registration,
             provider=provider,
+            parent=kwargs.get('parent')
```

Comment:
Should this be kwargs.get('parent', None')`?

Reply:
`get` intercepts Key errors and returns None by default. I'm cool being more explicit if that's the norm elsewhere

Reply:
I'm good with that. Thanks!